modules/ROOT/pages/using-policies.adoc

```diff
@@ -1,13 +1,13 @@
-= Applying a Policy
```

Comment:
Hi mighty Kim, can you please edit the PR for me:
https://github.com/mulesoft/docs-api-manager/pull/321
Type: CE
Scope: All
Due: Per your convenience